admin/dashboard/assets/src/dashboard-app/pages/fonts/edit/EditGoogleFont.js

```diff
@@ -75,42 +77,17 @@ const EditGoogleFont = ({fontId, fontName}) => {
 		}
 	});
 
+	useEffect( () => {
+		setTimeout(() => {
```

Comment:
Do we really require this setTImeout here ? @imnavanath

Reply:
@nathanrodrigues2111 Yes! @nathanrodrigues2111 We need this because when we hit the add button then, it takes some time to update as we are hitting API every time for autosave. so that's why we needed this.

Reply:
But would it cause issue on slower internet speeds ? @yssambare12

Reply:
So maybe we can reduce the timeout from 3 sec to 1.
Actually, if we remove this then Without this effect, the success message would remain visible until another action triggers a state change that updates fontUpdateAction.

Reply:
okay @yssambare12